corehq/apps/reports/forms.py

```diff
@@ -103,7 +104,11 @@ def clean(self):
 
 
 class ScheduledReportForm(forms.Form):
-    INTERVAL_CHOICES = [("daily", "Daily"), ("weekly", "Weekly"), ("monthly", "Monthly")]
+    INTERVAL_CHOICES = [
+        ("daily", ugettext("Daily")),
```

Comment:
Used `ugettext` instead of `ugettext_lazy` because the lazy version could not get json serialized due to a proxy that `ugettext_lazy` returns apparently (see [this](https://stackoverflow.com/a/24666856) SO comment).